doc/source/parameters/cfd/boundary_conditions_multiphysics.rst

```diff
@@ -0,0 +1,60 @@
+==================================
+Multiphysics - Boundary Contitions
+==================================
+
+This subsection's purpose is defining the boundary conditions associated to multiphysic problems. 
+
+Heat Transfer
+^^^^^^^^^^^^^
+
+For heat transfer boundary conditions, the defaults parameters are:
```

Comment:
```suggestion
For heat transfer boundary conditions, possible parameters are:
```

(the default value is only temperature = 0, no convection)

Reply:
the type parameter is set to temperature by default, but when using a convection type, these are the default parameters. Maybe we could rephrase that

Reply:
I rephrased it, but the branch was merged before I finished. I'll put it in the next PR as a bonus